app/templates/select_application.html

```diff
@@ -24,16 +24,18 @@
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-full">
             <span class="govuk-caption-m">{{ fund.short_name + " - " + fund.title_json["en"] }}</span>
-            <h1 class="govuk-heading-l">Select an application</h1>
+            <h1 class="govuk-heading-l govuk-!-margin-0">Select an application</h1>
+            <p class="govuk-body govuk-!-margin-top-1 govuk-!-margin-bottom-7">Select the application you want to manage or create a new one.</p>
+
         </div>
     </div>
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-full">
             <form method="POST" novalidate>
                 {{ form.csrf_token }}
-                {{ form.round_id }}
+                {{ form.round_id(params={"label":{"classes": "govuk-visually-hidden", "text": "Select the application you want to manage or create a new one."} })  }}
```

Comment:
Can we keep the content in the python form? We can reference it here if needed with `form.round_id.label.text`?